doc/environment.yml

```diff
@@ -16,3 +16,4 @@ dependencies:
 - sphinx
 - sphinx_rtd_theme
 - ipython=4.0.1
+- cartopy
```

Comment:
good point! though I think it'll need to be in a `pip` section because sphinx_gallery isn't available on conda